### ISSUE TITLE
[Bug]: Boot stuck at 30-35%. After applying post-install patches

### ISSUE THREAD

**Issue by CormacW43** [dortania/OpenCore-Legacy-Patcher#385]
### Machine Model

MacBookPro8,3

### Application Version

0.2.3 (Latest Build)

### Application Variant

TUI (Text User Interface)

### What versions of macOS are you seeing the problem on?

macOS 11, Big Sur

### Where does this issue happen?

Between booting macOS and Login Screen

### What is the Issue?

When turning on the MacBook Pro it starts up normally until it gets to the progress bar and Apple logo screen, after that it freezes at about 30-35% and refuses to go any higher. It is refusing to boot after I have applied the post-install patches.

![image](https://user-images.githubusercontent.com/87450584/125702360-5c148b91-f8c4-4a85-bd64-e8edbbd23547.jpg)

**Comment by genisgusvian** [comment]
i have same problem with u. but not make sense . i always stuck on 30-40% on apple logo when i charging the macbook pro 15 inch mid 2010. when i unplug the charging i can boot normally. you can try it

**Comment by ezserval** [comment]
I'm having the exact same problem. My machine is also a MacBook Pro 8,3.

**Comment by genisgusvian** [comment]
> I'm having the exact same problem. My machine is also a MacBook Pro 8,3.

Unplug the charger and try it. I can booting normaly when i unplug the charging ( just use battery power without charging cable). When i booting my mac and the charger plug on the mac i can't booting. Just stuck on 40%

**Comment by ezserval** [comment]
> > I'm having the exact same problem. My machine is also a MacBook Pro 8,3.
> 
> 
> 
> Unplug the charger and try it. I can booting normaly when i unplug the charging ( just use battery power without charging cable). When i booting my mac and the charger plug on the mac i can't booting. Just stuck on 40%

Will try this. I'm running High Sierra now but I'm going to reinstall OCLP and see if it works.

**Comment by CormacW43** [comment]
I believe the problem may be caused by the AMD dGPU kexts. When I reinstalled High Sierra it wouldn’t boot and get stuck at 30-35%. I managed to resolve the issue by removing one of the AMD dGPU Kexts through single user mode. After I did this it booted perfectly fine.

**Comment by ezserval** [comment]
> I believe the problem may be caused by the AMD dGPU kexts. When I reinstalled High Sierra it wouldn’t boot and get stuck at 30-35%. I managed to resolve the issue by removing one of the AMD dGPU Kexts through single user mode. After I did this it booted perfectly fine.

Last month I used OCLP to install Big Sur on my MBP. It worked perfectly fine, but I decided to go back to Catalina. Yesterday I decided to update to Big Sur again by making a clean install, but I got stuck at the boot screen. Then I wiped my SSD and tried to install High Sierra, but I also couldn't pass through the boot screen. Then I reseted NVRAM and SMC, disabled the dGPU again via Single User Mode and booted on Safe Boot Mode. Installed the MBP dGPU Disabler by dosdude1 and now it is working fine again.
